marine/marine.py

```diff
@@ -4,6 +4,8 @@
 from io import StringIO
 from typing import Optional, List, Dict
 
+from marine.encap_consts import *
```

Comment:
`from . import encap_consts`
Yes, it will be less comfortable, but it's preferred (in order to improve the static analysis of the code).
Also, that will allow you to shorten the encap names to just `ETHERNET` or `WIFI` (however I'm not sure we'd like to use such nicknames and maybe full on linktype names are preferred).
If that makes your usage awkward I'd suggest picking a shorter name for the module (maybe just `encap`).